src/main/java/net/sf/jabref/logic/citationstyle/CitationStyleGenerator.java

```diff
@@ -56,17 +56,18 @@ protected static String generateCitation(BibEntry entry, String style, CitationS
 
         } catch (IOException | ArrayIndexOutOfBoundsException e) {
             LOGGER.error("Could not generate BibEntry Citation", e);
+            return Localization.lang("Cannot generate Citation");
```

Comment:
I think, this generates the string displayed in the preview panel. Maybe, the message should be "Cannot generate preview based on selected citation style".

Further, "Citation" is not a fixed term, which should be uppercased. Put it lower case as most English words are lower case.